src/sass/vue2-animate.scss

```diff
@@ -8,6 +8,7 @@
  */
 
 $animationDuration: 1s !default;
+$quoteAnimationNames: true !default;
```

Comment:
Could you add a comment that reference to `https://github.com/asika32764/vue2-animate/pull/20/files` so developers can know how this variable works. Than you.